hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/condition/EventCondition.java

```diff
@@ -100,6 +118,7 @@ public boolean match(Event value) {
     private static final String ID = "id";
     private static final String CTIME = "ctime";
     private static final String TEXT = "text";
+    private static final String CATEGORY = "category";
     private static final String CONTEXT = "context.";
```

Comment:
I have a draft of a long response but at the end I think that this issue could have a better solution just limiting the inference against the tag map and as you suggest to not use context for it.
With proper documentation we can offer this and being uniform with Data and same use of the context map in other scenarios.